Know-Your-Nation-Speedy/NUnitTestKYNS/UnitTest.cs

```diff
@@ -68,11 +68,11 @@ private void Seed(MyDbContext _db)
         {
             var entries = new[]
             {
-                new Entry{ Name="Mpilo Mshengu",Email="mpilo@gmail.com",Password="1234"},
-                new Entry{ Name="Linda",Email="Linda@gmail.com",Password="4321"},
-                new Entry{ Name="Buhle",Email="Buhle@gmail.com",Password="4321"}
+                new Users{ Name="Mpilo Mshengu",Email="mpilo@gmail.com",Password="1234"},
```

Comment:
Passwords should always be hashed when put into a db. Never have a password saved in plain text. Even a simple sha1 hash is fine to start off with but it is required

Reply:
security is to be targeted in our next sprint